.yamato/all-urp_performance_boatattack.yml

```diff
@@ -33,12 +33,12 @@ URP_Performance_BoatAttack_trunk:
          rerun: on_new_revision
       -  path: .yamato/urp_performance_boatattack-win-dx12.yml#URP_Performance_BoatAttack_Win_DX12_performance_build_editmode_mono_Linear_trunk
          rerun: on_new_revision
+      -  path: .yamato/urp_performance_boatattack-osx-metal.yml#URP_Performance_BoatAttack_OSX_Metal_performance_build_editmode_mono_Linear_trunk
+         rerun: on_new_revision
       -  path: .yamato/urp_performance_boatattack-android-opengles3.yml#URP_Performance_BoatAttack_Android_OpenGLES3_performance_playmode_runtime_il2cpp_Linear_trunk
          rerun: on_new_revision
       -  path: .yamato/urp_performance_boatattack-android-opengles3.yml#URP_Performance_BoatAttack_Android_OpenGLES3_performance_build_editmode_il2cpp_Linear_trunk
          rerun: on_new_revision
-      -  path: .yamato/urp_performance_boatattack-android-vulkan.yml#URP_Performance_BoatAttack_Android_Vulkan_performance_playmode_runtime_il2cpp_Linear_trunk
```

Comment:
Is the Android job supposed to be removed?

Reply:
(I think it is since I see a commit explicitly mentioning it - just curious about the reason :))

Reply:
yep - mentioned in description